src/components/common/text-input.tsx

```diff
@@ -23,3 +25,22 @@ export const TextInput: FunctionComponent<ButtonProp> = ({ className, placeHolde
     />
   );
 };
+
+export const DataList: FunctionComponent<ButtonProp> = ({ placeHolder, onChange, dataTestId, options, value }) => {
+  const _onChange = (newValue: any): void => {
+    return onChange(newValue.value);
+  };
+
+  return (
+    <Creatable
```

Comment:
Can't use arrows

Reply:
What do u mean by can't use arrows

Reply:
try to navigate with arrows

Reply:
fixed. The issue is cause useEffect continuous looping recreating the dropdown. I have fixed it by running useEffect only once